src/backend/merger/main.go

```diff
@@ -104,6 +104,7 @@ func MergePdf(ctx context.Context) error {
 	return err
 }
 
+// getPort checking on port and returning
```

Comment:
the definition is not correct to replace it with returns port number for backend merger http server to listen default is 8080

Reply:
@shambhav15 edit is required